Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1490,6 +1491,12 @@
     <string name="statistics_explanation_faq_url">"https://www.coronawarn.app/en/faq/#further_details"</string>
     <!-- XACT: Statistics explanation illustration description -->
     <string name="statistics_explanation_illustration_description">"Abstract picture of a smartphone with four placeholders for information"</string>
+    <!-- XTXT: Statistics explanation blog text -->
+    <string name="statistics_explanation_blog">"Mehr inhaltliche Details zu einzelnen Kennzahlen finden Sie im Blog."</string>
+    <!-- XTXT: Statistics explanation blog text label -->
+    <string name="statistics_explanation_blog_label">"Blog."</string>
```

Comment:
Should the "." be part of the link? Or maybe only "blog"?

Reply:
I've thought about this, but the figma design has the "." part of the link so I've kept it like that :)

Reply:
alright, perfect!